tests/test_phreeqc.py

```diff
@@ -345,21 +345,52 @@ def test_alkalinity():
     OH = solution.get_amount("OH-", "mol/L").magnitude
     H = solution.get_amount("H+", "mol/L").magnitude
     # Alkalinity calculated from the excess of negative charges from weak acids
-    total_alk_molL = HCO3 + 2 * CO3 + OH - H
-    # Convert to mg/L as CaCO3
-    wt_caco3 = 100.09 / 2  # g/mol
-    total_alk_mgL = total_alk_molL * wt_caco3 * 1000  # mg/L
-    # alkalinity reported by mg/L, and if tried converting to mol/L yields ZeroDivisionError: float division by zero
-    assert alk.to("mg/L").magnitude == pytest.approx(total_alk_molL, abs=0.01)
+    total_alk = HCO3 + 2 * CO3 + OH - H
+    assert alk.to("mg/L").magnitude == pytest.approx(total_alk, abs=0.01)
 
-def test_equilibrate_unrecognized_component():
+
+def test_equilibrate_liquid():
     solution = Solution([["Cu+2", "4 mol/L"], ["O-2", "4 mol/L"]], volume="2 L", engine="phreeqc")
-    # Specifying an unrecognized solid raises an Exception
-    with pytest.raises(Exception):  # noqa: B017, PT011
-        solution.equilibrate(solids=["Ferroxite"])
+    solution.equilibrate()
+    assert solution.get_total_amount("Cu", "mol").magnitude == pytest.approx(1.654340558452752)
+
```

Comment:
Please change this test to Ca2+ instead of copper, and use a lower concentration like 0.05 M so that the Ca doesn't precipitate. That way the expected result (0.05 M * 2 L) will be much clearer